packages/plugins/error-tracking/package.json

```diff
@@ -1,15 +1,15 @@
 {
-    "name": "@dd/rum-plugin",
```

Comment:
it could be nice [mark the old package as deprecated](https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions) (maybe you're planning to do that already)

Reply:
these are not published 😉

Reply:
woops right 👍